newIDE/app/src/SceneEditor/index.js

```diff
@@ -1825,6 +1844,23 @@ export default class SceneEditor extends React.Component<Props, State> {
             </React.Fragment>
           )}
         </I18n>
+        {this.state.invisibleLayersOnWhichInstancesHaveJustBeenAdded && (
+          <InfoBar
+            message={
+              <Trans>
+                You just added an instance to the layers
```

Comment:
should we add a ⚠️  at the beginning?

Reply:
Can we add to multiple layers at the same time? 🤔 
I'd simplify the sentence to `You just added an instance to a hidden layer (xxx). Open the layer panel to make it visible`

Reply:
I say:
- I agree on simplifying the sentence (honestly, nobody will make the error multiple time, we warn about the last layer, it's good enough). And so probably simplify the state to contain the last layer.
  Less is more. With the plural, the sentences is getting hard to parse.
- No ⚠️. If we want to add this, it should be a prop and part of the design of the app, not an emoji.